src/mmw/package.json

```diff
@@ -29,6 +29,7 @@
     "jstify": "0.9.0",
     "leaflet": "0.7.3",
     "leaflet-draw": "0.2.3",
+    "leaflet.locatecontrol": "0.43.0",
```

Comment:
Please add new deps to vendor bundle as well by adding an entry here: https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/bundle.sh#L109